File: BeeCrowd Python/Iniciante/2061.py
def main():
  abas_iniciais,numero_acoes = map(int,input().split())
  abas = abas_iniciais

  for x in range(0,numero_acoes):
    acao = str(input())

    if acao == "fechou":
      abas += 1
    elif acao == "clicou":
      abas -= 1

  print(abas)

main()

File: BeeCrowd C++/Iniciante/2061.cpp
#include <iostream>
using namespace std;

int main(void){
  int abas,numero_acoes,abas_iniciais;
  string acao;
  
  cin >> abas_iniciais >> numero_acoes;
  abas = abas_iniciais;

  for (int i = 0; i < numero_acoes; i++){
    cin >> acao;

    if (acao == "clicou"){
      abas--;
    }else if (acao == "fechou"){
      abas++;
    }
  }

  cout << abas << endl;
  return 0;
}
